std/haxe/ds/IntMap.hx

```diff
@@ -57,11 +57,17 @@ extern class IntMap<T> implements haxe.Constraints.IMap<Int, T> {
 
 	/**
 		See `Map.keys`
+
+		(cs, java) Implementation detail: Do not `set()` any new value while
+		iterating, as it may cause a resize, which will break iteration.
```

Comment:
Note that modify-while-iterate is undefined behavior in general.

Reply:
Are there any tests for this? Are there any targets where it actually works fine?